contracts/Core/RewardManager.sol

```diff
@@ -9,6 +9,7 @@ import "./interface/ICollectionManager.sol";
 import "../Initializable.sol";
 import "./storage/Constants.sol";
 import "./parameters/child/RewardManagerParams.sol";
+import "hardhat/console.sol";
```

Comment:
hey it wont be included in final bytecode, as sol compiler would do tree shaking 
But even so, can we remove this 
Just for being clean sake

Reply:
All else LGTM

Reply:
removed https://github.com/razor-network/contracts/pull/674/commits/a67257c9ee441c2c86295869fe5ba7093dfbba58